codex-cli/scripts/run_in_container.sh

```diff
@@ -51,14 +68,21 @@ cleanup
 # Run the container with the specified directory mounted at the same path inside the container.
 docker run --name "$CONTAINER_NAME" -d \
   -e OPENAI_API_KEY \
+  -e ALLOWED_DOMAINS="$ALLOWED_DOMAINS" \
```

Comment:
I believe it is possible for `ALLOWED_DOMAINS` to be the empty string if this script were run with `ALLOWED_DOMAINS=` (explicitly defining the `ALLOWED_DOMAINS` env var as the empty string), so out of an abundance of caution, maybe we should do this before `docker run`:

```
if [ -z "$ALLOWED_DOMAINS" ]; then
  echo "Error: ALLOWED_DOMAINS is empty."
  exit 1
fi
```